pyproject.toml

```
@@ -15,7 +15,7 @@ write_to = "evm_trace/version.py"
 
 [tool.black]
 line-length = 100
-target-version = ['py38', 'py39', 'py310', 'py311', 'py312']
+target-version = ['py38', 'py39', 'py310', 'py311', 'py312', 'py313']
```

looks like 3.8 isnt supported

```suggestion
target-version = ['py39', 'py310', 'py311', 'py312', 'py313']
```